### ISSUE TITLE
(putty.install) Putty 0.78 installs only for current user due to installer 'bug'

### ISSUE THREAD

**Issue by pluto00987** [chocolatey-community/chocolatey-packages#2243]
### Chocolatey Version

2.0.0

### Chocolatey License

None

### Package Version

0.78

### Current Behaviour

Putty only installs for the current user (ie start menu icons, add/remove programs entry, etc are all specific to one user)

### Expected Behaviour

Putty should install for all users

### Steps To Reproduce

Run choco install putty.install
Log in with any other user and see no start menu icons or add/remove programs entry

### Environment

```markdown
- Operating System: Microsoft Windows NT 10.0.17763.0
- PowerShell Version: 5.1.17763.3770
- Shell: powershell
```

### Chocolatey Log

```shell
n/a
```

### Anything else?

This seems to be a known bug with the 0.78 installer, as detailed here:
https://www.chiark.greenend.org.uk/~sgtatham/putty/wishlist/win-install-scope.html

The suggested workaround seems to address the issue (add 'ALLUSERS=1' to the install command)

**Comment by pauby** [comment]
@pluto00987 Are you in a position to put a PR in for this?

**Comment by pluto00987** [comment]
The putty installer issue was fixed in v0.79. As the chocolatey package is already up to v0.80 now, this issue should be resolved.
